week-12/homework/index.js

```diff
@@ -0,0 +1,131 @@
+let input = document.querySelector('.input');
```

Comment:
Same for this JS file, the file formatting does not look right to me, let's sharing next time so I can fix up your Prettier